public_html/wp-content/mu-plugins/camptix-tweaks/addons/privacy.php

```diff
@@ -78,15 +79,17 @@ public function render_registration_field( $form_data, $i ) {
 			</td>
```

Comment:
Seems like the `tix-required-star` in the line above should also have `aria-hidden`.

Reply:
Thanks for pointing that out — I also noticed that the edit form used a different render function, so I've also gone through and updated the "Edit Ticket" fields too.